lib/manageiq/v2v/engine.rb

```diff
@@ -10,12 +10,15 @@ def self.vmdb_plugin?
       app.config.assets.paths << root.join('assets', 'images').to_s
     end
 
+    # TODO: Un-comment the analytics menu item below when we are ready to release it
+
     initializer 'plugin' do
       Menu::CustomLoader.register(
         Menu::Section.new(:migration, N_("Migration"), 'fa fa-plus', [
           Menu::Item.new('plans', N_("Migration Plans"), 'migration', {:feature => 'migration', :any => true}, '/migration#/plans'),
           Menu::Item.new('mappings', N_("Infrastructure Mappings"), 'mappings', {:feature => 'mappings', :any => true}, '/migration#/mappings'),
-          Menu::Item.new('settings', N_("Migration Settings"), 'migration_settings', {:feature => 'migration_settings', :any => true}, '/migration#/settings')
+          Menu::Item.new('settings', N_("Migration Settings"), 'migration_settings', {:feature => 'migration_settings', :any => true}, '/migration#/settings')# ,
+          # Menu::Item.new('analytics', N_("Migration Analytics"), 'migration_analytics', {:feature => 'migration_analytics', :any => true}, '/migration#/analytics')
```

Comment:
We're going to need a new product feature called `migration_analytics` for this new menu item. Is there a core PR for this somewhere? Or should I perhaps create one?

Reply:
Hmm, good point. I didn't create one, but I'm not sure if we ever created one for `mappings` or `migration_settings` either. Can you point me to where those are defined? we might be able to just have these all under the `mappings` feature. @fdupont-redhat do you know if we need these menu items to be under different CF features?

Reply:
This is where the current product features for v2v are defined: https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml#L1115

Reply:
oh weird, conversion hosts are listed under migration overview, they should be under migration settings

Reply:
Ok, sounds like we will need a `migration_analytics` product feature then. Do you mind making the PR for that @mzazrivec? good catch.

Reply:
@mturley Those `conversion_host*` product features were added b/c of API it seems. Perhaps we could revisit / fix this, since the yaml structure is also propagated into UI and can be a bit confusing.

Reply:
Just to illustrate my point above:

![roles](https://user-images.githubusercontent.com/6648365/59523017-70e66580-8ed0-11e9-8085-242d4c29bf58.png)

Reply:
Yeah, that is confusing, especially since it looks like those granular features can be turned off individually for certain roles. If the user can access the Migration Settings page but they can't use the conversion_hosts API endpoints, there will be errors there. Also, I don't do any kind of role/feature checking in the BreadcrumbSwitcher, it just renders links to all 4 pages no matter what. I wonder if that needs to change, and if so how we can do those checks from React...

Reply:
@mturley new product feature PR: https://github.com/ManageIQ/manageiq/pull/18877

Reply:
> Also, I don't do any kind of role/feature checking in the BreadcrumbSwitcher ...

Yeah, this can become a problem, should the role not have some of the migration features enabled (unlikely, but can happen).

To fix this in react, we'd have to maintain our map of `bc_item: feature` and pull enabled features via api:
```
# get user's group id first
$ /api/users/...
# get enabled product features for given group id
$ /api/groups/...?attributes=miq_product_features
```

Reply:
ok, thanks for that info. I'll open a separate issue to track that, we should try to get it in for 5.11 I think.

Reply:
@mzazrivec @mturley Product features are now pluggable, so it could live in this repo directly.

Reply:
@mzazrivec I opened https://github.com/ManageIQ/manageiq-v2v/issues/989 to track the breadcrumb switcher thing. @Fryguy that's good to know, we should probably move that stuff to this repo then. Is that documented somewhere? I'm not sure where that needs to live in this repo.

Reply:
Pluggable product features just landed in https://github.com/ManageIQ/manageiq/pull/18806.  No documentation yet.